examples/simple.yml

```
@@ -146,9 +146,19 @@ data-silos:
           - key: firstName
             title: First Name
             description: The first name of the user, inputted during onboarding
+            categories:
+              - category: USER_PROFILE
```

```suggestion
              - USER_PROFILE
```

can we just do this? feels weird to have two named fields like this

---

@michaelfarrell76 it's because we allow users to label things with their own subcategory:

```
categories:
    - category: CONTACT
       name: email
```